fault-monitor/operational-status-monitor.cpp

```diff
@@ -19,6 +19,46 @@ namespace status
 {
 namespace monitor
 {
+void Monitor::removeCriticalAssociation(const std::string& objectPath)
+{
+    try
+    {
+        PropertyValue getAssociationValue = dBusHandler.getProperty(
+            objectPath.c_str(), "xyz.openbmc_project.Association.Definitions",
+            "Associations");
+
+        auto association = std::get<AssociationsProperty>(getAssociationValue);
```

Comment:
This can throw "std::bad_variant_access"? Do you need to handle that?

Reply:
done